.github/workflows/ci.yml

```diff
@@ -22,32 +22,50 @@ jobs:
         with:
           dotnet-version: 7.0.x
       - name: Set up Node.js
-        uses: actions/setup-node@v4.0.1
+        uses: actions/setup-node@v4
         with:
-            node-version: 16
+          node-version: latest
```

Comment:
do we want to use latest and allow the build failing to signal breaking node version changes, or would it be better to set the current latest version number and have each release responsible for attempting to upgrade to latest?

Reply:
I couldn't really find any documentation talking about needing to check the CI/CD workflows to make sure that their action's versions are up to date. I'm not really sure if there is a process in place for maintaining GitHub workflows and the versions. It feels like these versions could get stale pretty fast.  To me, it seems to make sense that we would want to make sure that our code is compatible with the latest versions of node, but I do acknowledge that having broken builds because of a node version update could be disruptive to the development cycle and dependabot PRs for bumping version in C# projects.

However, I'm also not opposed to having a set version to increase stability.  But maybe we could update the [releasing guidelines](https://github.com/Particular/Platform/blob/main/guidelines/releasing.md) or the [release workflow](https://github.com/Particular/Platform/blob/bce1aeb559c6147d542a02b482fd114b206cfd7f/guidelines/github-actions/release-workflow.md) documentation to indicate that when releasing a new version, the GitHub actions should be checked?